### ISSUE TITLE
Issues about the Reference sequence track error

### ISSUE THREAD

**Issue by 211melody** [GMOD/jbrowse-components#4461]
I got this error from JBrowse, here is the stack trace:

```
Error loading source map, showing raw stack trace below:
ReferenceError: Buffer is not defined
ReferenceError: Buffer is not defined
    at BgzipIndexedFasta._fetchFromIndexEntry (http://unpkg.com/@jbrowse/react-linear-genome-view/dist/react-linear-genome-view.umd.development.js:28074:26)
    at BgzipIndexedFasta.getResiduesByName (http://unpkg.com/@jbrowse/react-linear-genome-view/dist/react-linear-genome-view.umd.development.js:28055:21)
```

**Comment by cmdcolin** [comment]
thanks for reporting this. i think i figured out a fix, sort of similar to this PR but apply it to the indexedfasta https://github.com/GMOD/bam-js/pull/98

**Comment by cmdcolin** [comment]
@211melody I think this should be fixed now in v2.12.2 if you want to check

**Comment by 211melody** [comment]
> @211melody I think this should be fixed now in v2.12.2 if you want to check
Yes, it is ok now, thanks

**Comment by cmdcolin** [comment]
awesome, glad that worked :)
